spec/features/form/check_answers_page_lettings_logs_spec.rb

```diff
@@ -159,26 +159,6 @@
       end
     end
 
-    context "when viewing setup section answers" do
-      before do
-        FactoryBot.create(:location, scheme:, startdate: Time.zone.local(2021, 1, 1))
-      end
-
-      it "displays inferred postcode with the location id" do
-        lettings_log.update!(location:)
-        visit("/lettings-logs/#{id}/setup/check-answers")
-        expect(page).to have_content("Location")
-        expect(page).to have_content(location.name)
-      end
-
-      it "displays inferred postcode with the location_admin_district" do
-        lettings_log.update!(location:)
-        visit("/lettings-logs/#{id}/setup/check-answers")
-        expect(page).to have_content("Location")
-        expect(page).to have_content(location.location_admin_district)
-      end
-    end
-
```

Comment:
Do we just want to update this test with translation instead of removing it? It doesn't look like it's only testing content

Reply:
These tests are for 2021, which won't have a translation file. We could update the tests to use 2023 going forward?

Reply:
I guess we could where it makes sense to